dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityServiceTest.java

```diff
@@ -601,6 +601,34 @@ void shouldReturnTrackedEntitiesGivenUserHasDataReadAccessToTrackedEntityType()
         trackedEntities);
   }
 
+  @Test
+  void shouldReturnTrackedEntitiesWithCorrectOrgUnitPropertiesMappedWhenTrackedEntityAccessible()
+      throws ForbiddenException, NotFoundException, BadRequestException {
+    TrackedEntityOperationParams operationParams =
```

Comment:
Can we test

> Currently we don't set the path value, so access to a TE is always rejected.

AFAIK this test only tests the mapping which is not all that useful IMHO. This test also runs as the import user which I am not sure is a super user 🤔

Reply:
The user running this test is not a super user.

Here I'm testing the TE that's returned from the API call, it's not just a mapping, this is the data we store in the database.
If the TE is returned, it already validates it's no longer rejected.
On top of that, I'm making sure the org unit path is set, which was the reason why it was rejected before.